### ISSUE TITLE
NTR: Endoscopy: gastric folds markedly enlarged (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8660]
New term request Endoscopy: gastric folds markedly enlarged (nadea3_200213183053): Giant mucosal folds  UMLS: 0940258

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000249
- label: Enlarged gastric folds
- parents: [Abnormal stomach morphology (HP:0002577)](https://hpo.jax.org/app/browse/term/HP:0002577)
- pmid: PMID:20966317
- def: Giant gastric folds are recognized at endoscopy as an increase in the size of the gastric rugae and a lack of flattening.
- comment: The most common causes of giant gastric folds are Menetrier disease, acute gastric mucosal lesions, gastric lymphoma, and scirrhous carcinoma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8660
- synonyms: Giant gastric folds
